### PR TITLE
[MAINT] Add Javier Gonzalez-Castillo to Zenodo file

### PR DESCRIPTION
**Pull request by javiergcas**
This pull request includes my information for the .zenodo.json file to be added as a contributor

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -30,6 +30,11 @@
       "affiliation": "National Institutes of Health",
       "orcid": "0000-0001-7261-4042"
     }
```
Comment (tsalo) [inline .zenodo.json:32]:
```suggestion
    },
```

**Review by tsalo — CHANGES_REQUESTED** [review]
Just needs a comma in the entry before you to be valid. 

Also, since our release notes come from the PR titles, do you mind if I change the title to 
`"[MAINT] Add Javier Gonzalez-Castillo to Zenodo file"`?

**Comment by javiergcas** [comment]
Hi Taylor.

That is fine. What do I need to do? Open a new pull request and add the
comma?

Javier.

**Comment by jbteves** [comment]
@javiergcas I'll walk you through it when you get a chance this week, we can message about it separately.

**Review by jbteves — APPROVED** [review]
LGTM, thanks @javiergcas !

**Review by tsalo — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**